crashreports/crashreporter/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+set(TARGET crashreporter)
```

Comment:
Again, a long comment at top.

This builds a separate executable from Audacity, which is a sister of the Audacity object file in the installed tree.  So I think it should be at the same level of the source tree.  Is there any reason that would not work?

So I think top CMakeLists might first visit this, which defines the crashreporter target.

The crashreports library is in effect dependent on this target.  Not really for build purposes, but at runtime it assumes this executable is present to be called.

Perhaps that dependency should be encoded in the CMake relations, by having this target have a compile_define in its INTERFACE (such as, the string literal "crashreporter" that I mention elsewhere).